tokio feature 'rt' required for building

While building bluez-async v0.5.1, I encountered the following errors:
`error[E0432]: unresolved import 'tokio::task::JoinError'`
`error[E0425]: cannot find function 'spawn' in crate 'tokio'`

After inspecting the cargo.toml I found that in `tokio v1.0.1`, which is used by the most recent bluez-async, both the import and the function specified by the errors require the feature 'rt'. However, this function is only under the dev-dependencies for tokio, causing both errors:

`[dependencies.tokio]
version = "1.0.1"`

`[dev-dependencies.tokio]
version = "1.0.1"
features = ["macros", "rt", "rt-multi-thread", "time"]`

How to fix:
Specify 'rt' as a feature with tokio, under the normal dependencies.

(This is my first time opening an issue on GitHub, and I didn't find any guidelines for creating an issue, so please leave constructive feedback if possible.)

--- a/bluez-async/Cargo.toml
+++ b/bluez-async/Cargo.toml
@@ -22,7 +22,7 @@ serde = "1.0.118"
 serde_derive = "1.0.118"
 serde-xml-rs = "0.5.0"
 thiserror = "1.0.23"
-tokio = "1.0.1"
+tokio = { version = "1.0.1", features = ["rt"] }
 uuid = "0.8.1"
 
 [dev-dependencies]

